charts/function-mesh-operator/values.yaml

```diff
@@ -59,3 +59,12 @@ controllerManager:
   # resourceLabels: {}
   # resource annotations applied to each function/connector managed by this controller
   # resourceAnnotations: {}
+  webhook:
+    configFile: /etc/config/config.yaml
```

Comment:
the configFile is not for webhook, please check https://github.com/streamnative/function-mesh/pull/322 for more info.